game/kernel/kmemcard.cpp

```diff
@@ -58,11 +59,11 @@ using namespace ee;
 template <typename... Args>
 void mc_print(const std::string& str, Args&&... args) {
   if (memcard_debug) {
-    fmt::print("[MC] ");
+    lg::info("[MC] ");
```

Comment:
will this have issues with newlines? Does this print even get used?

Reply:
Not anymore since `memcard_debug` is turned off, but it does mess up newlines. It's meant to look lime `[MC] message goes here`